torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6373,15 +6426,6 @@ def method_tests():
         ('squeeze', (S, 1, S, 1), (1,), '1_dim', (True,), [0]),
         ('squeeze', (S, 1, S, 1), (2,), 'not_1_dim', (True,), [0]),
         ('squeeze', (), (0,), 'scalar', (True,), [0]),
-        ('split', (S, S, S), (2,), '', (True,)),
-        ('split', (S, S, S), (S, 1), 'dim', (True,), [1]),
-        ('split', (S, S, S), ([int(S / 3), S - int(S / 3) * 2, int(S / 3)],), 'size_list',
-            (True, 'aten::split_with_sizes')),
-        ('split', (S, S, S), ([int(S / 2), S - int(S / 2) * 2, int(S / 2)], 2), 'size_list_dim',
-            (True, 'aten::split_with_sizes'), [1]),
-        ('split_with_sizes', (S, S, S), ([int(S / 3), S - int(S / 3) * 2, int(S / 3)],), '', (True,)),
-        ('split_with_sizes', (S, S, S), ([int(S / 3), S - int(S / 3), 0],), 'size_0', (True, )),
-        ('split_with_sizes', (S, S, S), ([int(S / 3), S - int(S / 3) * 2, int(S / 3)],), 'dim', (True, ), [1]),
```

Comment:
I assume that this cases was supposed to add a dim as well to the argument. Have added one in `sample_inputs_split_with_sizes`.

Reply:
I think the test generator from method_tests understood the dim argument and would programmatically generate some options for it

Reply:
I don't think that is the case.
For checking, I had added the following snippet after the linked line.
```python
if 'split_with' in test_name:
      print(test_name, new_args)
```
https://github.com/pytorch/pytorch/blob/82d714935ea7c54f9e5966f47adf4b8460eb2c3d/test/test_autograd.py#L5390

Output for the given sample was (not all args are same),
```
test_split_with_sizes_dim_neg0 ([1, 3, 1],)
test_split_with_sizes_dim ([1, 3, 1],)
test_split_with_sizes_dim_neg0_complex ([1, 3, 1],)
test_split_with_sizes_dim_complex ([1, 3, 1],)
```

Reply:
Interesting; thanks for investigating. Good thing soon method_tests() will be gone and we won't have to wonder at its mysteries anymore